README.md

```diff
@@ -1,10 +1,12 @@
 # ONNX Model Zoo
+[![Build Status](https://img.shields.io/azure-devops/build/onnx-pipelines/onnx/11?label=Windows&logo=azure-pipelines)](https://dev.azure.com/onnx-pipelines/onnx/_build?definitionId=11)
+[![Build Status](https://img.shields.io/azure-devops/build/onnx-pipelines/onnx/11?label=Linux&logo=azure-pipelines)](https://dev.azure.com/onnx-pipelines/onnx/_build?definitionId=10)
 
 [Open Neural Network Exchange (ONNX)](http://onnx.ai) is an open standard format for representing machine learning models. ONNX is supported by a community of partners who have implemented it in many frameworks and tools.
 
 The ONNX Model Zoo is a collection of pre-trained, state-of-the-art models in the ONNX format contributed by community members like you. Accompanying each model are [Jupyter notebooks](http://jupyter.org) for model training and running inference with the trained model. The notebooks are written in Python and include links to the training dataset as well as references to the original paper that describes the model architecture.
 
-We are standardizing on [Git LFS (Large File Storage)](https://git-lfs.github.com/) to store our ONNX model files. To download an ONNX model, navigate to the appropriate Github page and click the `Download` button on the top right.
+We have standardized on [Git LFS (Large File Storage)](https://git-lfs.github.com/) to store our ONNX model files. To download an ONNX model, navigate to the appropriate Github page and click the `Download` button on the top right.
```

Comment:
nit: rem "our" -> to store ONNX model files instead of to store our ONNX model files?

Reply:
Fixed. Thanks!